.gitignore

```diff
@@ -18,3 +18,4 @@ MANIFEST-*
 OPTIONS-*
 CURRENT
 LOCK
+*/CACHEDIR.TAG
```

Comment:
I think we should keep this tag it might help us later with github actions to check if they were any changes in the repo.